source/AMDSClient.h

```diff
@@ -56,6 +56,7 @@ private slots:
 	QSpinBox* amptekIndex;
 	QComboBox *bufferNameComboBox_;
 	QCheckBox* includeStatusDataCheckbox;
```

Comment:
All of these internal variables should have a trailing _ to ensure that they don't have conflicts with other names.